.github/workflows/docker.yml

```diff
@@ -38,18 +46,20 @@ jobs:
       name: Checkout
       uses: actions/checkout@v2
     -
-      if: ${{ needs.check_secrets.outputs.DOCKERHUB_USERNAME == 'true' }}
+      if: ${{ needs.check_secrets.outputs.DOCKER_BASE_REPO == 'true' }}
       name: Docker meta
       id: docker_meta
       uses: crazy-max/ghaction-docker-meta@v2
       with:
         images: |
-          ${{ secrets.DOCKERHUB_USERNAME }}/spid-compliant-certificates
+          ${{ secrets.DOCKER_BASE_REPO }}/spid-compliant-certificates
```

Comment:
We don't need this to be a secret, we can just use the string "italia/spid-compliant-certificates"

Reply:
It's required to allow workflow testing also in forks. In my case, `DOCKERHUB_USERNAME` and `DOCKER_BASE_REPO` have the same value. This make possible the publishing of [`psmiraglia/spid-compliant-certificates`](https://hub.docker.com/r/psmiraglia/spid-compliant-certificates) image.